packages/run-protocol/src/vaultFactory/liquidateIncrementally.js

```diff
@@ -0,0 +1,322 @@
+/* eslint-disable no-await-in-loop */
+// no @ts-check
+
+import { E } from '@endo/eventual-send';
+import {
+  getAmountOut,
+  makeRatio,
+  assertProposalShape,
+  offerTo,
+  natSafeMath as NatMath,
+} from '@agoric/zoe/src/contractSupport/index.js';
+import { AmountMath } from '@agoric/ertp';
+import { Far } from '@endo/marshal';
+
+import { ceilMultiplyBy } from '@agoric/zoe/src/contractSupport/ratio';
+import { makeTracer } from '../makeTracer.js';
+// import { quote } from '@agoric/assert';
+
+const { details: X } = assert;
+const trace = makeTracer('LiqI');
+
+/**
```

Comment:
consider putting this at the top as a `/** @file` comment.

Reply:
It is at the top already. I'll add `@file`

Reply:
I meant tippy top, above imports, but under imports is fine